ios/COVIDSafePaths.xcodeproj/project.pbxproj

```diff
@@ -2130,9 +2130,9 @@
 					"-ObjC",
 					"-lc++",
 				);
-				PRODUCT_BUNDLE_IDENTIFIER = "org.pathcheck.covid-safepaths";
```

Comment:
To check, were these changes meant to be bundled into this PR?

Reply:
nope!